modules/ROOT/nav.adoc

```diff
@@ -1,6 +1,6 @@
 * xref:home:index.adoc[Docs Home]
 * xref:get-started:index.adoc[Get Started]
-** xref:get-started:whats-new.adoc[]
+** xref:get-started:whats-new-241.adoc[]
```

Comment:
@micheleRP I guess we are missing topic-aware leader balancing

Reply:
@mattschumpert correct, we were not planning doc for that for beta. See [here](https://redpandadata.atlassian.net/wiki/spaces/DOC/pages/531234904/Core+Version+24.1).

Reply:
All good @micheleRP. ship it!